src/components/comments/CommentItem/CommentContent.tsx

```
@@ -0,0 +1,122 @@
+    return (
+        <div className="mb-2">
+            <div
+                className="text-sm text-gray-700 dark:text-gray-300 whitespace-pre-wrap break-words prose prose-sm max-w-none dark:prose-invert"
+                style={{ 
+                    resize: 'none',
+                    overflow: 'visible',
+                    display: 'block',
+                    minHeight: 'auto',
+                    height: 'auto'
+                }}
+                dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)